CONTRIBUTING.md

```diff
@@ -117,6 +117,78 @@ You can learn more about [pull requests on GitHub](https://help.github.com/en/gi
 
 You can also contribute to the project without specifically writing content for the site. Many discussions are happening all around the project. Protocol Labs, and by extension, IPFS, is a collaborative company that relies heavily on the community. We need your opinions and input! If you have an idea about a topic or want to weigh in on a discussion, feel free to have your say either on [GitHub](https://github.com/ipfs/ipfs-docs/issues) or on the [IPFS forums](https://discuss.ipfs.tech/).
 
+## Create a new page
+
+Sometimes you will need to make a whole new page for content to live. The process to do this is outlined below.
+
+These steps were performed in Visual Studio Code (VSCode).
+
+It is important to know what kind of content you are creating. The questions below will help to determine where your new content should live. For example, you wouldn't want to put a page written for experienced users between the already existing **Install** and **Basics** sections.
+
+- What type of content am I creating?
+- Who is my intended audience?
+- Does this content require a new page or can it be added to an existing page?
+- If a new page or section must be added, how will it fit into the existing structure and flow of the docs?
+
+When you have the above questions figured out and you know where the new content will live, continue on with the steps below.
```

Comment:
Phrasing: "When you have the above questions..." is probably not the easiest way to word this.

Suggestion: Something like "When you've answered the questions above..."